app/page.tsx

```diff
@@ -2,100 +2,56 @@ import Image from "next/image";
 
 export default function Home() {
   return (
-    <div className="grid grid-rows-[20px_1fr_20px] items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-20 font-[family-name:var(--font-geist-sans)]">
-      <main className="flex flex-col gap-8 row-start-2 items-center sm:items-start">
-        <Image
-          className="dark:invert"
-          src="/next.svg"
-          alt="Next.js logo"
-          width={180}
-          height={38}
-          priority
-        />
-        <ol className="list-inside list-decimal text-sm text-center sm:text-left font-[family-name:var(--font-geist-mono)]">
-          <li className="mb-2">
-            Get started by editing{" "}
-            <code className="bg-black/[.05] dark:bg-white/[.06] px-1 py-0.5 rounded font-semibold">
-              app/page.tsx
-            </code>
-            .
-          </li>
-          <li>Save and see your changes instantly.</li>
-        </ol>
+    <div className="min-h-screen p-8 pb-20 gap-16 sm:p-20 font-[family-name:var(--font-geist-sans)]">
+      <main className="max-w-4xl mx-auto">
+        <div className="text-center space-y-6 mb-12">
+          <h1 className="text-4xl sm:text-6xl font-bold tracking-tight">
+            Your Dog's Personal Chauffeur
+          </h1>
+          <p className="text-xl text-muted-foreground">
+            Safe, reliable transportation for your four-legged family members.
+            From vet visits to grooming appointments, we've got your pup
+            covered.
+          </p>
+        </div>
+
+        <div className="grid grid-cols-1 sm:grid-cols-2 gap-8 mb-16">
+          <div className="p-6 border rounded-lg">
+            <h2 className="text-2xl font-semibold mb-4">For Pet Parents</h2>
+            <ul className="space-y-3">
+              <li>✓ Real-time tracking of your dog's journey</li>
+              <li>✓ Vetted and trained pet transport specialists</li>
+              <li>✓ Flexible scheduling for routine appointments</li>
+              <li>✓ Safe, comfortable, climate-controlled vehicles</li>
+            </ul>
+          </div>
+
+          <div className="p-6 border rounded-lg">
+            <h2 className="text-2xl font-semibold mb-4">How It Works</h2>
+            <ul className="space-y-3">
+              <li>1. Book a ride through our easy-to-use app</li>
+              <li>2. Meet your certified pet transport specialist</li>
```

Comment:
those class names are a mouthful 💀 be smart and use something like BEM or SMACSS for a cleaner and more maintainable CSS architecture